src/language/sharedLanguage.ts

```diff
@@ -72,7 +72,11 @@ const parserOptions: HTMLReactParserOptions = {
      * Replace p tag with Paragraph component from design system
      */
     if (isElement(domNode) && domNode.name === 'p') {
-      return React.createElement(Paragraph, { size: 'sm' }, domToReact(domNode.children as DOMNode[], parserOptions));
+      return React.createElement(
```

Comment:
Hvorfor bruker man `React.createElement` i stedet for TSX her egentlig? 😄

Reply:
Tja, ikke noen spesiell grunn egentlig, fila var opprinnelig .ts i guess 🤷‍♂️

Reply:
Jeg merger nå, men det er bare å endre det hvis du vil rydde opp 🙌